src/controller/jobRoleController.ts

```diff
@@ -0,0 +1,16 @@
+import type { Request, Response } from "express";
+import type { JobRole } from "../model/jobRole";
+import { getJobRoles } from "../service/jobRoleService";
+
+export namespace JobRoles {
+    export async function get(req: Request, res: Response): Promise<void> {
+        try {
+            const jobRoles: JobRole[] = await getJobRoles();
+            res.locals.jobRoles = jobRoles;
```

Comment:
I don't mind if you use res.locals or pass the data in via the res.render method as long as it is consistent for the team. Can you make sure everyone is on the same page